mitosheet/src/mito/components/endo/ColumnHeaderDropdown.tsx

```diff
@@ -39,188 +38,66 @@ export default function ColumnHeaderDropdown(props: {
         }
     }, [props.display])
         
-
-    // If the user interacts with the column header dropdown, we always default to 
-    // selecting the first row for any action, like setting the column formula. 
-    const rowIndex = 0 
-    const columnIndex = Object.keys(props.sheetData.columnIDsMap).indexOf(props.columnID)
+    const getPropsForDropdownItem = (actionName: ActionEnum) => {
+        const action = props.actions.buildTimeActions[actionName];
+        return {
+            title: action.toolbarTitle ?? action.longTitle,
+            onClick: () => {
+                props.closeOpenEditingPopups();
+                void action.actionFunction();
+            },
+            icon: action.contextMenuIcon ? <action.contextMenuIcon /> : action.toolbarIcon ? <action.toolbarIcon/> : <StepsIcon/>,
```

Comment:
Let's change these to iconContextMenu and iconToolbar, it makes them easier to parse, I think. 

TiP: you can just do a `fn + F2` on the actual Action type, and you'll be able to do this find and replace super duper easy!